src/modules/LoggedIn/Project/ModalPage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Button, Modal } from 'antd'
+const ModalPage = () => {
```

Comment:
Component `ModalPage` ini direname jadi `ProjectForm` ya, biar lebih jelas tujuannya

```suggestion
const ProjectForm = () => {
```